pkg/controllers/user/pipeline/controller/pipeline/pipeline.go

```diff
@@ -48,8 +48,9 @@ func (l *Lifecycle) Updated(obj *v3.Pipeline) (runtime.Object, error) {
 
 func (l *Lifecycle) Remove(obj *v3.Pipeline) (runtime.Object, error) {
 	if obj.Status.WebHookID != "" {
-		if err := l.deleteHook(obj); err != nil && !errors.IsNotFound(err) {
-			return obj, err
+		if err := l.deleteHook(obj); err != nil {
+			logrus.Warnf("fail to delete webhook for pipeline %q: %v", obj.Name, err)
```

Comment:
I would like to use `logrus.WithError(err).Warnf(xxx)`

Reply:
Updated